services/register.js

```diff
@@ -0,0 +1,47 @@
+const AWS = require('aws-sdk');
+AWS.config.update({
+    region: 'us-east-1'
```

Comment:
I think for this, it would be better if we use us-west-2 region, to keep things consistent. Also, I believe this config can be reused, you can shift it to utils folder and add a new file there named constants and export a dictionary? We can then pass the dictionary as AWS.config.update(AWS_CONFIG).

Reply:
Done.